src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,128 @@
+
+  const sortByField = searchParams.get(SortQueries.sort);
+  const sortOrder = searchParams.get(SortQueries.order) || SortOrder.off;
+
+  if (!filteredPeople.length) {
+    return;
+  }
+
+  const sortHandler = (sortField: SortBy) => {
+    let tempSortOrder = SortOrder.off;
+
+    if (sortField === sortByField) {
+      switch (sortOrder) {
+        case SortOrder.off:
+          tempSortOrder = SortOrder.asc;
+          break;
+        case SortOrder.asc:
+          tempSortOrder = SortOrder.decs;
+          break;
+      }
+    } else {
+      tempSortOrder = SortOrder.asc;
+    }
+
+    const newParams = new URLSearchParams(searchParams);
+
+    newParams.set(SortQueries.sort, sortField);
+    newParams.set(SortQueries.order, tempSortOrder);
+
+    navigate({
+      search: newParams.toString(),
+    });
+  };
+
+  const getClasses = (field: SortBy) => {
+    let arrowPos = 'fa-sort';
+
+    if (sortByField === field) {
+      if (sortOrder === SortOrder.asc) {
+        arrowPos = 'fa-sort-up';
+      }
+
+      if (sortOrder === SortOrder.decs) {
+        arrowPos = 'fa-sort-down';
+      }
+    }
+
+    return 'fas ' + arrowPos;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Name
+              <a onClick={() => sortHandler(SortBy.name)}>
+                <span className="icon">
+                  <i className={getClasses(SortBy.name)} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Sex
+              <a onClick={() => sortHandler(SortBy.sex)}>
+                <span className="icon">
+                  <i className={getClasses(SortBy.sex)} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Born
+              <a onClick={() => sortHandler(SortBy.born)}>
+                <span className="icon">
+                  <i className={getClasses(SortBy.born)} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Died
+              <a onClick={() => sortHandler(SortBy.died)}>
+                <span className="icon">
+                  <i className={getClasses(SortBy.died)} />
+                </span>
+              </a>
+            </span>
+          </th>
+
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {sortPeople(filteredPeople, sortByField as keyof Person, sortOrder).map(
+          person => (
+            <PersonItem key={person.slug} person={person} />
+          ),
+        )}
```

Comment:
You are sorting the people every time the component is re-rendered. This could lead to performance issues if the list of people is large. Consider using useMemo to store the sorted list.